lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -682,18 +689,22 @@ network
     => ApiLayer s t k
     -> Handler ApiNetworkInformation
 network ctx = do
-    nTip <- liftHandler (networkTip nl)
     now <- liftIO getCurrentTime
+    nodeTip <- liftHandler (NW.networkTip nl)
+    let ntrkTip = fromMaybe slotMinBound (slotAt sp now)
```

Comment:
just nitpicking - maybe `netTip` would be more immediate what it is ;-)

Reply:
Neither you nor Jonathan likes this abbreviation for network :'( .. I think I'll just stop using it.